proto/japella/japellaapi/v1/japella.proto

```
@@ -1,7 +1,9 @@
~syntax = "proto3";
 
 option go_package = "./gen/protobuf";
 
+package japella.japellaapi.v1;
```

Buf (1.47.2)

Files with package "japella.japellaapi.v1" must be within a directory "japella/japellaapi/v1" relative to root but were in directory "proto/japella/japellaapi/v1".

(PACKAGE_DIRECTORY_MATCH)